### PR TITLE
Go CLI PoC: Add importer logic

### PR DESCRIPTION
**Pull request by ArthurSens**

### CODE REVIEW

File: installer/cmd/render.go [hunk 1]
```diff
@@ -30,6 +31,14 @@ var renderCmd = &cobra.Command{
...
 			fmt.Println(item)
 		}
+		mixinImporter := importer.NewMixinImporter("https://github.com/gitpod-io/observability", "")
+		mixinImporter.ImportPrometheusRules()
+
+		yamlImporter := importer.NewYAMLImporter("https://github.com/ArthurSens/observability", "manifests/production/meta/kubescape", false)
+		yamlImporter.Import()
+
+		kustomizeImporter := importer.NewKustomizeImporter("https://github.com/kubernetes-sigs/kustomize", "examples/helloWorld")
+		kustomizeImporter.Import()
```
Comment (meysholdt) [inline installer/cmd/render.go]: if the imports are just examples and we don't actually want to import the stuff - would it be better to uncomment them?

Comment (ArthurSens) [reply installer/cmd/render.go]: Do you mean comment? Yep that can be done 🙂

Comment (meysholdt) [reply installer/cmd/render.go]: yes - comment :)

Comment (ArthurSens) [reply installer/cmd/render.go]: done!

**Review by meysholdt — APPROVED** [review]
🚀

**Pull request merged**